docs/extending-overseerr/fail2ban.md

```diff
@@ -0,0 +1,93 @@
+# Protecting Your Server Using Fail2ban
+
+{% hint style="warning" %}
+If your OS runs `firewalld`, make sure it is running and that Overseerr is accessible before continuing.
+{% endhint %}
+
+### Configuring Fail2ban
+
+After installing Fail2ban, the configuration files should be located at `/etc/fail2ban`. Make ***copies*** of `jail.conf` and `fail2ban.conf` in the same directoy, replacing the `.conf` extensions with `.local`.
+
+Next, open `jail.local` in a text editor.  Add the following jail configuration under `JAILS`:
+
+```
+[overseerr]
+enabled         = true
+port            = 5055
+logpath         = /root/snap/overseerr/common/logs/overseerr.log
```

Comment:
```suggestion
logpath         = /path/to/overseerr.log
```

Reply:
Ooh good catch!  Can't believe I missed that when proofreading :|